gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java

```diff
@@ -94,10 +90,21 @@ private InternalLoggingOptions(LoggingOptions options) {
     protected ExecutorFactory<ScheduledExecutorService> executorFactory() {
       return super.executorFactory();
     }
+
+    @Override
+    protected ApiCallSettings.Builder apiCallSettings() {
```

Comment:
`InternalLoggingOptions` extends `LoggingOptions` so that `protected` methods inherited from `LoggingOptions` can be made accessible in a different package (as `DefaultLoggingRpc` is in `com.google.cloud.logging.spi` while `LoggingOptions` is in `com.google.cloud.logging`). The alternative to this would be much such methods in `LoggingOptions` `public`, which we don't want to do (yet).